seatunnel-api/src/main/java/org/apache/seatunnel/api/table/type/VectorType.java

```diff
@@ -30,8 +30,8 @@ public class VectorType<T> implements SeaTunnelDataType<T> {
     public static final VectorType<Map> VECTOR_SPARSE_FLOAT_TYPE =
             new VectorType<>(Map.class, SqlType.SPARSE_FLOAT_VECTOR);
 
-    public static final VectorType<Byte> VECTOR_BINARY_TYPE =
-            new VectorType<>(Byte.class, SqlType.BINARY_VECTOR);
+    public static final VectorType<ByteBuffer> VECTOR_BINARY_TYPE =
```

Comment:
Is this change compatible with older versions?

Reply:
Currently first connector of binary vectors is implemented in this pr with no historical version problem